src/main/java/pl/sda/awesomemovies/client/movie/MovieClientService.java

```diff
@@ -1,41 +1,29 @@
 package pl.sda.awesomemovies.client.movie;
 
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
-import org.springframework.web.client.ResourceAccessException;
 
-import java.util.Arrays;
-import java.util.Collections;
+import java.util.ArrayList;
 import java.util.List;
 
 @Service
 public class MovieClientService {
 
-    private final MovieRestService movieRestService;
-
-    @Autowired
-    public MovieClientService(MovieRestService movieRestService) {
-        this.movieRestService = movieRestService;
-    }
+    private List<Movie> moviesList = new ArrayList<>();
 
     public List<Movie> getAllMovies() {
-        try {
-            Movie[] movies = movieRestService.provide().getForObject(
-                    movieRestService.getEndpointUrl() + "/movies",
-                    Movie[].class);
-            return Arrays.asList(movies);
-        } catch (ResourceAccessException e) {
-            return Collections.emptyList();
-        }
+        moviesList.add(new Movie(1L, "Star Wars"));
```

Comment:
Przy każdym odpaleniu listy będzie się zwiększała lista filmów czy to oczekiwany efekt? :-)
Jeżeli nie to można zrobić statyczny blok na górze:

```
static {
        moviesList.add(new Movie(1L, "Star Wars"));
        moviesList.add(new Movie(2L, "Star Trek"));
        moviesList.add(new Movie(3L, "Oblivion"));
        moviesList.add(new Movie(4L, "Avengers"));
}
```
Wtedy zostanie to wywołane raz podczas inicjalizacji klasy dokładnie raz.